src/components/ec-donut/ec-donut.vue

```diff
@@ -0,0 +1,169 @@
+<template>
+  <div class="ec-donut">
+    <div class="ec-donut__container">
+      <svg
+        width="108"
+        height="108"
+        viewBox="0 0 108 108"
+        class="ec-donut__svg"
+      >
+        <circle
+          class="ec-donut__background"
+          cx="54"
+          cy="54"
+          :r="radius"
+          fill="transparent"
+          stroke-width="11"
+        />
+        <circle
+          class="ec-donut__remaining"
+          cx="54"
+          cy="54"
+          :r="radius"
+          fill="transparent"
+          stroke-width="11"
+          :stroke-dasharray="dashArray"
+          :stroke-dashoffset="0"
+        />
+        <circle
+          class="ec-donut__used"
+          cx="54"
+          cy="54"
+          :r="radius"
+          fill="transparent"
+          stroke-width="11"
+          :stroke-dasharray="dashArray"
+          :stroke-dashoffset="offset"
+        />
+      </svg>
+    </div>
+    <div class="ec-donut__legend">
+      <div class="ec-donut__legend-used">
+        <ec-icon
+          name="rounded-notification"
+          :size="14"
+          class="ec-donut__legend-icon ec-donut__legend-icon--used"
+        />
+        <slot name="used-legend" />
+      </div>
+      <div class="ec-donut__legend-reminder">
+        <ec-icon
+          name="rounded-notification"
+          :size="14"
+          class="ec-donut__legend-icon ec-donut__legend-icon--reminder"
+        />
+        <slot name="reminder-legend" />
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import EcIcon from '../ec-icon';
+
+export default {
+  name: 'EcDonut',
+  components: { EcIcon },
+  props: {
+    amount: {
+      type: Number,
+      required: true,
+    },
+    used: {
+      type: Number,
+      required: true,
+    },
+  },
+  data() {
+    return {
+      radius: 48,
+    };
+  },
+  computed: {
+    percentageUsed() {
+      return (this.used / this.amount) * 100;
```

Comment:
We should handle the case that used is greater than amount, so it never goes above 100%.